archaius2-core/src/main/java/com/netflix/archaius/ProxyFactory.java

```diff
@@ -68,6 +70,24 @@ private String derivePrefix(Configuration annot, String prefix) {
         return prefix + ".";
     }
     
+    /**
+     * Populate a set of all interface and sub interface methods via recursion.
+     * Give top level overrides priority over methods defined in super-interfaces
+     * @param type
+     * @param methods
+     */
+    private void fillConfigMethods(Class<?> type, Set<Method> methods) {
```

Comment:
True.  I was following a pattern from governator.  I need to understand why governator didn't use getMethods().